sensor/kubernetes/certrefresh/certrepo/service_certificates_repository.go

```diff
@@ -205,14 +209,14 @@ type patchSecretDataByteMap struct {
 }
 
 func (r *ServiceCertificatesRepoSecrets) createSecret(ctx context.Context, caPem []byte,
-	certificate *storage.TypedServiceCertificate, secretSpec ServiceCertSecretSpec) (*v1.Secret, error) {
+	certificate *storage.TypedServiceCertificate, secretSpec ServiceCertSecretSpec, desiredLabels map[string]string) (*v1.Secret, error) {
```

Comment:
Here, I think you can "hardcode" the TLS labels instead of passing them as `desiredLabels`, because this will always be a tls secret. You could rename this function `createTlsSecret` to convey this.